clients/src/test/java/org/apache/kafka/common/metrics/MetricsTest.java

```diff
@@ -424,6 +424,14 @@ public void testPercentiles() {
         assertEquals(0.0, p25.value(), 1.0);
         assertEquals(0.0, p50.value(), 1.0);
         assertEquals(0.0, p75.value(), 1.0);
+
+        // record two more windows worth of sequential values
```

Comment:
Hmm, it seems that we need to advance the mocked time to force the rolling of the old window?